### ISSUE TITLE
web: dashboard: when ListAllJobs is disabled, allow showing job data on-demand per-pipeline

### ISSUE THREAD

**Issue by jamieklassen** [concourse/concourse#5556]
## Summary

When the endpoint is disabled, let's make the sync icon clickable. Clicking it should hit the `ListJobs` endpoint for the pipeline in question and update the pipeline card. For now, this will be a one-time-only operation -- it will not start a loop of polling for that pipeline's data, nor will it be possible to do another manual sync without fully refreshing the page... Accordingly, that data will not get persisted into the localStorage cache?

![variations master](https://user-images.githubusercontent.com/6372810/79888100-fdec2c00-83c9-11ea-8f07-b6893d54178a.png)

For now, let's track the tooltip as part of this work too.

## Context

* Depends on #5470
* Part of #5405

**Comment by aoldershaw** [comment]
Can we make the sync button spin when the sync is in process? That would be SWEET

**Comment by taylorsilva** [comment]
# Issue Bankruptcy

This issue is being closed as part of 2025's **Issue Bankruptcy** declaration. [See here for details](https://blog.concourse-ci.org/posts/2025-01-06-concourse-2024-review/)

If you think this issue still has merit then feel free to open a new issue and reference this one.
